packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -175,6 +176,9 @@ export const FeedContainer = ({
           aria-live={subject === ToastSubject.Feed ? 'assertive' : 'off'}
           data-testid="posts-feed"
         >
+          {router.query.welcome === '1' && (
```

Comment:
No strong opinion, I think in the past we used `true` as values.
Moving forward I suggest we pick one to go with, you favor 1?

Reply:
We can do true, really no benefits I see aside from URL length, but users never look at that. Moved to true.